app/kuma-dp/cmd/run.go

```diff
@@ -226,6 +203,37 @@ func newRunCmd(opts kuma_cmd.RunCmdOpts, rootCtx *RootContext) *cobra.Command {
 			}
 			opts.AdminPort = bootstrap.GetAdmin().GetAddress().GetSocketAddress().GetPortValue()
 
+			if cfg.DNS.Enabled && !cfg.Dataplane.IsZoneProxy() {
+				if cfg.DNS.CoreDNSConfigTemplatePath == "" && len(kumaSidecarConfiguration.Networking.CorefileTemplate) > 0 {
+					templateFile, err := writeTempFile("kuma-dp-corefile-template-", "Corefile", kumaSidecarConfiguration.Networking.CorefileTemplate, 0o644)
+					if err != nil {
+						return errors.Errorf("Failed to write corefile template to disk. %v", err)
```

Comment:
this is "nit", but errors shoudl be lowercase `failed to write...`. Check all errors in this PR for it.

Reply:
Thanks, will update code in an upcoming commit.